s3torchconnector/src/s3torchconnector/dcp/s3_file_system.py

```diff
@@ -196,24 +196,24 @@ class S3StorageWriter(FileSystemWriter):
     def __init__(
         self,
         region: str,
-        uri: str,
+        path: str,
```

Comment:
nit: I'd have stick to URI here, because they make more sense with S3, which does not really "think" in terms of path

Reply:
+1